pkg/controllers/controlplanemachineset/watch_filters_test.go

```diff
@@ -91,6 +79,52 @@ var _ = Describe("Watch Filters", func() {
 		}
 	}
 
+	Context("filterClusterOperator", func() {
+		const operatorName = "control-plane-machine-set"
+
+		var clusterOperatorPredicate predicate.Predicate
+
+		BeforeEach(func() {
+			clusterOperatorPredicate = filterClusterOperator(operatorName)
+		})
+
+		It("Panics with the wrong object kind", func() {
+			expectedMessage := "expected to get an of object of type configv1.ClusterOperator"
+			machine := resourcebuilder.Machine().Build()
+			Expect(func() {
```

Comment:
Nit: Please include a new line between the declarations and expectations
```suggestion
			machine := resourcebuilder.Machine().Build()
			
			Expect(func() {
```